### PR TITLE
refactor tests

### PR DESCRIPTION
**Pull request by JoranAngevaare**
## What does the code in this PR do / what does it improve?
Using #1022 cleanup some of the plugin - related tests

We loose a bit of coverage, mostly of code that is very much duplicated and/or intended as testing only - see e.g. the setting up of hitfinder thresholds.

## Can you briefly describe how it works?
Please see #1022

### CODE REVIEW

File: straxen/plugins/event_processing.py [hunk 1]
```diff
@@ -224,22 +224,21 @@ def _set_dtype_requirements(self):
...
+            ('n_competing', np.int32, 'number of competing peaks'),
+            ('max_pmt', np.int16, 'PMT number which contributes the most PE'),
+            ('max_pmt_area', np.float32, 'area in the largest-contributing PMT (PE)'),
+            ('range_50p_area', np.float32, 'width, 50% area [ns]'),
+            ('range_90p_area', np.float32, 'width, 90% area [ns]'),
+            ('rise_time', np.float32, 'time between 10% and 50% area quantiles [ns]'),
             ('area_fraction_top', np.float32, 'fraction of area seen by the top PMT array'),
             ('tight_coincidence', np.int16, 'Channel within tight range of mean'),
-            ('n_saturated_channels',      np.int16, 'Total number of saturated channels'),
+            ('n_saturated_channels', np.int16, 'Total number of saturated channels'),
```
Comment (JoranAngevaare) [inline straxen/plugins/event_processing.py:241]: This was codefactor starting to complain about this for some reason..

File: straxen/test_utils.py [hunk 2]
```diff
@@ -16,58 +14,6 @@
...
-    hit_min_amplitude_mv=80,
-    hit_min_amplitude_he='pmt_commissioning_initial_he',
-    avg_se_gain=1.0,
-    se_gain=1.0,
-    rel_extraction_eff=1.0,
-    rel_light_yield=1.0,
-    g1=0.1,
-    g2=10,
-    bayes_config_file=f'resource://{aux_repo +"2df37896923eb9ca3157befb6274cb697ac2f4f7/strax_files/dummy_bayes_model.npz"}?fmt=npy',
-)
```
Comment (JoranAngevaare) [inline straxen/test_utils.py:70]: No more testing with very unclear configs / infinite re-iterations to make the test works 🎉

File: straxen/plugins/peak_processing.py [hunk 3]
```diff
@@ -181,7 +181,7 @@ def compute(self, peaks_he):
             (first_sr1_run, pax_file('XENON1T_tensorflow_nn_pos_weights_20171217_sr1.h5'))]),   # noqa
     strax.Option('min_reconstruction_area',
                  help='Skip reconstruction if area (PE) is less than this',
-                 default=10,  infer_type=False,),
+                 default=10, infer_type=False, ),
```
Comment (JoranAngevaare) [inline straxen/plugins/peak_processing.py:184]: codefactor...

**Review by WenzDaniel — APPROVED** [review]
Thanks a lot as a lot of changes, but as far as I can tell everything looks fine.

**Pull request merged**